src/queues/sqs/mod.rs

```diff
@@ -222,6 +222,6 @@ impl From<DeleteMessageError> for AppError {
 
 impl From<JsonError> for AppError {
     fn from(error: JsonError) -> AppError {
-        AppErrorKind::QueueError(format!("JSON error: {:?}", error)).into()
+        AppErrorKind::JsonError(format!("JSON error: {:?}", error)).into()
```

Comment:
Changed this to something more broadly applicable so that I could re-use it in the `db` module.